crates/typst-library/src/foundations/str.rs

```diff
@@ -793,6 +817,18 @@ cast! {
     v: Str => Self::Str(v),
 }
 
+#[derive(PartialEq, Cast)]
+pub enum UnicodeNormalForm {
+    #[string("NFC")]
```

Comment:
the usual case convention for Typst string enums would be all lowercase, i.e. `"nfc"`.